### PR TITLE
atf: Reorganize

### PR DESCRIPTION
**Pull request by guochunrong**
*Add FUSE DISTRO_FEATURES
*Add chassistype and ddrphyopt
*Install srk.pri and srk.pub

Signed-off-by: Chunrong Guo <chunrong.guo@nxp.com>

### CODE REVIEW

File: recipes-bsp/atf/atf_git.bb [hunk 1]
```diff
@@ -2,6 +2,7 @@ DESCRIPTION = "ARM Trusted Firmware"
 
 LICENSE = "BSD"
 LIC_FILES_CHKSUM = "file://license.rst;md5=e927e02bca647e14efd87e9e914b2443"
+PR = "r1"
```
Comment (otavio) [inline recipes-bsp/atf/atf_git.bb]: Please drop, as this is not needed.

**Comment by guochunrong** [comment]
done

**Comment by otavio** [comment]
Please squash the commits so we keep history clean.

**Comment by guochunrong** [comment]
resubmit

**Pull request closed without merging**